lib/beaker-rspec/helpers/serverspec.rb

```diff
@@ -109,7 +108,6 @@ def self.method_missing(meth, *args)
     end
 
     def self.run(meth, *args)
-      backend = Specinfra.backend
```

Comment:
https://github.com/voxpupuli/beaker-rspec/pull/144 also takes an axe to this method.